### ISSUE TITLE
Problem with `root` parameter 

### ISSUE THREAD

**Issue by randaalex** [tmuxinator/tmuxinator#488]
`root` option work only for first windows (because in changed manually with `cd` command).
the second window doesn't ignore change directory option.

Looks like a problem in my tmux. It has -c option, but ignore it (I tried to run command manually, and directory has not changed).
```
...
$ tmux new-window -c /vagrant/api/pr2 -t pr:1 -n api/pr2                                                            
$ tmux new-window --help                                                                                                                  
usage: new-window [-adkP] [-c start-directory] [-F format] [-n window-name] [-t target-window] [command]
```

Perhaps we can stop using `-c` parameter for non-first windows, and change directory manually (with `cd` command), the same as in a first window?

I have tmuxinator from master branch, ubuntu 14.04, tmux 2.0 (also tried 1.8).
My ~/.tmux.conf empty now
My .tmuxinator.yml looks like:
```
name: project
root: /vagrant

windows:
  - api/pr1:
      root: /vagrant/api/pr1
      panes:
        - rails s

  - api/pr2:
      root: /vagrant/api/pr2
      pre: nvm use v0.10.25
      panes:
        - node main
```
My tmuxinator `project.render` looks like:
```
#!/bin/zsh

# Clear rbenv variables before starting tmux
unset RBENV_VERSION
unset RBENV_DIR

tmux start-server;

  cd /vagrant

  # Run pre command.


  # Create the session and the first window. Manually switch to root
  # directory if required to support tmux < 1.9
  TMUX= tmux new-session -d -s pr -n api/pr1
  tmux send-keys -t pr:0 cd\ /vagrant/api/pr1 C-m


  # Create other windows.
  tmux new-window -c /vagrant/api/pr2 -t pr:1 -n api/pr2


  # Window "api/pr2"
  tmux send-keys -t pr:0.0 rails\ s C-m

  tmux select-layout -t pr:0 tiled

  tmux select-layout -t pr:0
  tmux select-pane -t pr:0.0


  # Window "api/pr2"
  tmux send-keys -t pr:1.0 nvm\ use\ v0.10.25 C-m
  tmux send-keys -t pr:1.0 node\ main C-m

  tmux select-layout -t pr:1 tiled

  tmux select-layout -t pr:1
  tmux select-pane -t pr:1.0


  tmux select-window -t 0
  tmux select-pane -t 0

  if [ -z "$TMUX" ]; then
    tmux -u attach-session -t pr
  else
    tmux -u switch-client -t pr
  fi

```

**Comment by ethagnawl** [comment]
@randaalex I believe this is actually an issue with the whitespace in your YAML.

Try using the following (note the four space indentation before the panel config entries):

```
# ~/.tmuxinator/wat.yml
name: project
root: /vagrant

windows:
  - api/pr1:
      root: /vagrant/api/pr1
      panes:
        - rails s

  - api/pr2:
      root: /vagrant/api/pr1
      pre: nvm use v6.9.0
      panes:
        - node main
```

This has come up [in the past](https://github.com/tmuxinator/tmuxinator/issues/329) and I'm not sure what the root cause is - your config is valid YAML - but it's worth investigating further. I will create a separate issue for this. /cc @adamstrickland

Anyways, please close _this_ issue if the above solves your problem. Apologies for the inconvenience.

**Comment by randaalex** [comment]
@ethagnawl I sure YAML config totally valid. 
Because I get correct commands from `project.render` call (I used binding.pry for getting this value):
```
...
 # Create other windows.
  tmux new-window -c /vagrant/api/pr2 -t pr:1 -n api/pr2
```

I guess I have problem in my tmux. And my idea: refactor new-window dir changing from `-c new/path` to `tmux send-keys -t pr:0 cd\ /new/path C-m`. This let to remove a code for change workflow for old tmux versions (https://github.com/tmuxinator/tmuxinator/blob/master/lib/tmuxinator/assets/template.erb#L15-L25)

**Comment by ethagnawl** [comment]
@randaalex You're right - it's not a whitespace issue. Though, it appears to be an issue which only affects tmux > 2.0. (I was able to reproduce it using tmux 2.3.)

**Comment by josefson** [comment]
So, the following setup:
```
name: test
root: /Users/josefson/
windows:
  - editor:
      layout: main-vertical
      panes:
        - vim
        - top
  - server:
      panes:
          - echo 'server'
          - echo 'server 1'
  - logs:
      panes:
          - echo 'logs'
          - echo 'logs 1'
```
Outputs me the following and only opens the 1st window
```
**invalid option: default-path**
can't find pane 2
can't find window 2
can't find window 2
can't find window 2
can't find window 2
can't find window 2
can't find window 2
can't find window 3
can't find window 3
can't find window 3
can't find window 3
can't find window 3
can't find window 3
```
Although it does cd into the :root folder, it does not create the other windows. If i comment the :root statement however, it opens every window, but i lose the ability to specify a :root folder.
Does anybody knows any work around it?

**Comment by bennypowers** [comment]
FWIW I don't have root in any of my windows or panes:

```
# ~/.tmuxinator/eve.yml 
name: eve
root: /Users/bennyp/Documents/elastic/elastic-video-editor
 
windows:
  - editor:
      layout: main-vertical
      panes:
        - vim
        - yarn start
        - git lg
```

**Comment by dominikduda** [comment]
Any further info on this?

**Comment by mrtnmgs** [comment]
I'm having the same issue, is there a known workaround?

**Comment by fddayan** [comment]
+1
